lib/phlex.rb

```diff
@@ -7,6 +7,16 @@
 loader.ignore("#{__dir__}/generators")
 loader.setup
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.0")
+  unless Symbol.instance_methods.include?(:name)
+    class Symbol
+      def name
+        to_s.freeze
```

Comment:
This is slightly slower than if we just aliased `name` to `to_s`, but it’s a more faithful counterpart to the `Symbol#name` method in Ruby >= 3.0.